src/main/kotlin/blackjack/domain/betting/BettingResult.kt

```diff
@@ -0,0 +1,32 @@
+package blackjack.domain.betting
+
+import blackjack.domain.participant.Player
+import blackjack.domain.result.BlackjackResult
+import blackjack.domain.result.ResultType
+
+class BettingResult(private val bettingResult: Map<Player, Int>) {
```

Comment:
```suggestion
class BettingResult private constructor(private val bettingResult: Map<Player, Int>) {
```
팩토리 메서드를 활용한다면 생성자를 통한 객체 생성을 막도록 개선해보면 어떨까요?